src/components/Map.js

```diff
@@ -0,0 +1,3 @@
+export default function Map() {
```

Comment:
You could maybe have these titles directly in your Header and save these functions for your main content. But if how you did it works for you then it is great!

Reply:
Do you mean the components? or what? 🤔

Reply:
instead of creating a new component for each route, you could create a component for your header and in there define each route. would clean up your App.js and a Header is a great possibility to use as a component.

```
     <Route path="/map">
            <h2>map</h2>
       </Route>
```

Reply:
thank you John. I'll do that, makes more sense.